crates/sui-indexer-alt-graphql/src/pagination.rs

```diff
@@ -289,252 +289,3 @@ pub(crate) fn is_connection(field: &MetaField) -> bool {
     let type_ = field.ty.as_str();
     type_.ends_with("Connection") || type_.ends_with("Connection!")
 }
-
```

Comment:
Looks like you got rid of some of the existing tests here.

Reply:
Yes, thank you. Will try to be more vigilant this as well.